src/main/resources/templates/image-resizer.html

```diff
@@ -0,0 +1,85 @@
+﻿<!doctype html>
+<html xmlns:th="http://www.thymeleaf.org" th:replace="~{util/layout::base-page('Resize Image', ~{::content})}">
+<meta charset="UTF-8">
+<body>
+<main th:fragment="content">
+  <h2>Image Resizer Tool</h2>
+  
+  <div class="tool-description">
```

Comment:
Don't add classes that aren't used for anything. Lots of other cases of either non-existant classes, or tailwind classes when we're not using it. Please get rid of unused classes or add styles to them.